index.html

```diff
@@ -0,0 +1,168 @@
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+            <div class="card-work-item">
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="about">
+        <div class="about-me">
+            <h2>About Me</h2>
+            <p> I can help you build a product, feature or website Look through some of my work and experience! If you like
+                what you see and have a project you need coded,
+                don't hestiate to contact me.</p>
+            <div class="resume-btn"><a href="#">Get My Resume</a></div>
+        </div>
+        <div class="about-me-skils">
+            <div class="worklanguages">
+                <img src="./images/icon-languages.svg" alt="Image depicting languages">
+                <h2>LANGUAGES</h2>
+                <div class="lang-tags">
+                    <ul>
+                       <li>HTML</li>
+                       <li>Ruby</li>
+                       <li>JavaScript</li>
+                       <li>CSS</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="frameworks">
+                <img src="./images/icon-frameworks.svg" alt="Image that represent frameworks">
+                <h2>FRAMEWORKS</h2>
+                <div class="frameworks-tags">
+                    <ul>
+                       <li>Bootstrap</li>
+                       <li>Ruby On Rails</li>
+                       <li>RSpec</li>
+                       <li>Copybara</li>
+                       <li>Selenium</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="skills">
+                <img src="./images/icon-skills.svg" alt="An image that shows the skills">
+                <h2>SKILLS</h2>
+                <div class="skills-tags">
+                    <ul>
+                       <li>Codekit</li>
+                       <li>GitHub</li>
+                       <li>CodePen</li>
+                       <li>Gitlab</li>
+                       <li>Terminal</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </section>
```

Comment:
Great UI! 👍 However, there is a small issue with the vertical scrolling on the page. As per the design, there should be no vertical scrollbar visible. To fix this, you need to ensure that the content on the page fits within the viewport without requiring scrolling. 

[Portfolio (1).webm](https://github.com/zhorabay/My-portfolio/assets/55840999/5387f3a2-af2c-4ecf-913f-3bda0fbd7386)

Please review the content, layout, and sizing of the elements to ensure they are properly adjusted to eliminate the vertical scrolling. By addressing this issue, you will achieve a more polished and seamless user experience. Keep up the good work! 💯